plugins/shared-react/package.json

```diff
@@ -23,7 +23,9 @@
   },
   "dependencies": {
     "@kubernetes/client-node": "^0.18.1",
-    "classnames": "2.3.2"
+    "classnames": "2.3.2",
```

Comment:
shall we take changes in shared-react as separate PR and once released then rebase this?

Reply:
https://github.com/janus-idp/backstage-plugins/pull/633